integrationtests/utils.go

```diff
@@ -27,14 +33,22 @@ func createESClient(url string) (elasticproc.DatabaseClientHandler, error) {
 	})
 }
 
+//nolint
+func decodeAddress(address string) []byte {
+	decoded, err := pubKeyConverter.Decode(address)
+	log.LogIfError(err, "cannot decode address")
```

Comment:
"cannot decode address" won't be seen as the logger expects pairs of key-value after the error. so, remove "cannot decode address" or change to:
log.LogIfError(err, "address", address)

Reply:
changed